src/align.cpp

```diff
@@ -2072,7 +2067,7 @@ static void align_oc_msg_colon(chunk_t *so)
    {
       tmp = nas.m_aligned.GetChunk(idx);
 
-      tlen = tmp->str.size();
+      int tlen = tmp->str.size();
```

Comment:
Yes I think there are still places where a variable/parameter could be made const or the type might be better changed. But I didn't want to do that in this change request to avoid too many changes. 
This CR is growing rather larger anyway.